index.html

```diff
@@ -1,33 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio</title>
-  <script src="https://kit.fontawesome.com/9dfa895fe8.js" crossorigin="anonymous"></script>
+  <title>Portfolio Project</title>
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" href="style.css">
 </head>
-
 <body>
   <header class="navbar">
+    <div class="nav-logo">
+      <a  class="logo" href="#">My logo</a>
+    </div>
     <nav class="container">
-      <a class="logo" href="">My Logo</a>
-      <img src="images/Union.jpg" alt="Union">
+      <div class="menu-wrapper">
+        <img class="mobile-menu" src="images/Union.jpg" alt="union">
+      </div>
+      <ul class="nav-list">
+        <li class="nav-item"><a href="#portfolio">Portfolio</a></li>
+        <li class="nav-item"><a href="#about">about</a></li>
+        <li class="nav-item"><a href="#contact">contact</a></li>
+      </ul>
     </nav>
   </header>
-  <section class="hero">
-    <div class="container">
-      <h2 class="primary-text">I’m Ilham <br> Glade to see you!</h2>
-    </div>
-    <p class="secondary-text">
+<section id="portfolio" class="hero">
+  <div class="container">
+    <h2 class="primary-text">I’m Ilham Bouaik <br> Glade to see you!</h2>
+    <p class="secondary-text"> 
```

Comment:
- [ ] Please change the color of the heading to `#091E42`  as provided in Fimga. 
- [ ] The background image of this section looks blurry, I recommend using a better quality image for it, I downloaded it for you first one is svg, second png feel free to use them: 

![Header bg](https://user-images.githubusercontent.com/77038610/161224517-0fcff1cf-019a-461a-a911-58eba705b813.svg)
<img width="1398" alt="Header bg@2x" src="https://user-images.githubusercontent.com/77038610/161224664-ea5280e1-5354-478b-976c-e52b682e9a7a.png">